plugins/module_utils/common_ionos_module.py

```diff
@@ -34,6 +36,18 @@ def _should_update_object(self, existing_object, clients):
         """
         pass
 
+    def calculate_object_diff(self, existing_object, clients):
+        """
+        Calculate before and after for the object, only used in diff mode
+
+        existing_object : Ionoscloud object returned by API object
+        clients: authenticated ionoscloud clients list.
+
+        Returns:
+            dict, a dict with 2 keys: 'before' and 'after' which should only the attributes watched by ansible in their states
```

Comment:
not sure what "which should only the attributes" would mean here.
Maybe "which should only compare/contain the attributes"?

Reply:
it was "which shows only" in my head, I will change it